app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -303,7 +317,10 @@ class NewMessageFragment : Fragment() {
         toolbar.setNavigationOnClickListener { activity?.onBackPressedDispatcher?.onBackPressed() }
         changeToolbarColorOnScroll(toolbar, compositionNestedScrollView)
 
-        signatureWebView.enableAlgorithmicDarkening(true)
+        signatureWebView.apply {
+            isFocusable = false
+            enableAlgorithmicDarkening(true)
+        }
```

Comment:
See if you can replace all the isFocusable that are set in the code with focusable="false" + focusableInTouchMode="false" inside the XML. (Also you were missing the comment inside the code here that explains why this isFocusable is not set inside the XML directly)

Reply:
Work on Webview but not in the others isFocusable = false